pages/contact/index.vue

```diff
@@ -2,7 +2,7 @@
   <main class="text-center text-light-50">
     <div id="head-map" class="flex justify-center items-center">
       <Map class="w-full" />
-      <span class="absolute text-light-50 text-6xl xl:text-8xl font-bold font-display">Contact Us</span>
+        <span class="map-title absolute text-light-50 text-6xl xl:text-8xl font-bold font-display">Contact Us</span>
```

Comment:
Add other font-size classes to this span so the heading is responsive for small screen sizes as well. Currently, for mobile screens, the heading (and the new bg added) occupies much of the space making it difficult to appreciate the map behind.

Reply:
Added responsive classes in the latest commit.